.github/workflows/develop.yml

```diff
@@ -161,16 +161,20 @@ jobs:
         uses: Swatinem/rust-cache@v2
       - name: Install latest nextest release
         uses: taiki-e/install-action@nextest
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
       - name: Install Python
         uses: actions/setup-python@v4
         with:
           python-version: '3.10'
       - name: Install PyArrow Package
         run: pip install pyarrow
-      - name: Install cargo-llvm-cov
-        uses: taiki-e/install-action@cargo-llvm-cov
-      - name: Collect coverage data
-        run: cargo llvm-cov nextest --workspace --lcov --output-path lcov.info -F pyo3_backend -F dashboard
+      - name: Setup etcd server
+        working-directory: tests-integration/fixtures/etcd
+        run: docker compose -f docker-compose-standalone.yml up -d --wait
+      - name: Run nextest cases
+        run: cargo llvm-cov nextest --workspace --lcov --output-path lcov.info \
+                -F pyo3_backend -F dashboard --test-threads 1
```

Comment:
> --test-threads 1

Related to https://github.com/GreptimeTeam/greptimedb/issues/2357#issuecomment-1715471937. We should run tests on etcd with isolated prefix, but I don't find `EtcdStore` support such an prefix now.

Shall we implement this function first? @MichaelScofield @WenyXu

Reply:
Yes, we should.